lib/cog/command/service_sup.ex

```diff
@@ -12,8 +11,10 @@ defmodule Cog.Command.Service.Supervisor do
     token_monitor_table  = :ets.new(:token_monitor_table,  [:public])
     memory_table         = :ets.new(:memory_table,         [:public])
     memory_monitor_table = :ets.new(:memory_monitor_table, [:public])
+    data_path            = Application.get_env(:cog, Cog.Command.Service, [])[:data_path]
```

Comment:
I updated it to use:

```
Application.fetch_env!(:cog, Cog.Command.Service)
|> Keyword.fetch!(:data_path)
```

Which should raise if either `Cog.Command.Service` or the `data_path` key under it are missing from the Cog application configuration.